docs/PowerAuth-Server-0.24.0.md

```diff
@@ -5,36 +5,61 @@ This guide contains instructions for migration from PowerAuth Server version `0.
 ## Database Changes
 
 Following DB changes occurred between version 0.23.0 and 0.24.0:
-- Table `pa_activation` - added column `device_info`
+- Table `pa_activation` - added columns `device_info`, `platform`, `activation_otp`, `activation_otp_validation`.
 
 Migration script for Oracle:
 
 ```sql
 ALTER TABLE "PA_ACTIVATION" ADD "PLATFORM" VARCHAR2(255 CHAR);
 ALTER TABLE "PA_ACTIVATION" ADD "DEVICE_INFO" VARCHAR2(255 CHAR);
+ALTER TABLE "PA_ACTIVATION" ADD "ACTIVATION_OTP" VARCHAR2(255 CHAR);
+ALTER TABLE "PA_ACTIVATION" ADD "ACTIVATION_OTP_VALIDATION" NUMBER(2,0) DEFAULT 0 NOT NULL;
 ```
 
 Migration script for MySQL:
 
 ```sql
 ALTER TABLE `pa_activation` ADD `platform` varchar(255);
 ALTER TABLE `pa_activation` ADD `device_info` varchar(255);
+ALTER TABLE `pa_activation` ADD `activation_otp` varchar(255);
+ALTER TABLE `pa_activation` ADD `activation_otp_validation` int DEFAULT 0 NOT NULL;
 ```
 
 Migration script for PostgreSQL:
 
 ```sql
 ALTER TABLE "pa_activation" ADD "platform" VARCHAR(255);
 ALTER TABLE "pa_activation" ADD "device_info" VARCHAR(255);
+ALTER TABLE "pa_activation" ADD "activation_otp" VARCHAR(255);
+ALTER TABLE `pa_activation` ADD `activation_otp_validation` INTEGER DEFAULT 0 NOT NULL;
 ```
 
 ## Service Interface Changes
 
+PowerAuth server in version `0.24.0` slightly changed SOAP interface for protocol version `3` (namespace `http://getlime.io/security/powerauth/v3`):
+
+### Support for Additional Activation OTP
+
+- Added new enumeration `ActivationOtpValidation` with following values:
+  - `NONE` – no additional OTP validation is required during the activation.
+  - `ON_KEYS_EXCHANGE` – an additional OTP is validated during the keys-exchange activation phase.
```

Comment:
Can we use `key exchange` instead of `keys exchange`? See: https://en.wikipedia.org/wiki/Key_exchange

Reply:
OK, I also renamed that enum to ON_KEY_EXCHANGE :)